tools/ete/homology_classifier.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="homology_classifier" name="Homology Classifier and Filter" version="@VERSION@.1">
+    <description>from a genetree utilising the ETE Toolkit</description>
+    <macros>
+        <import>ete_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <stdio>
+        <!-- Anything other than zero is an error -->
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+    <![CDATA[
+python '$__tool_directory__/homology_classifier.py'
+--genetree '$genetreeFile'
+--out_format '$format_type.out_format'
+--filters $format_type.homologies
+> $homology
+    ]]>
+    </command>
+    <inputs>
+        <param name="genetreeFile" type="data" format="nhx" label="GeneTree file" help="GeneTree in nhx format, where nodes are in form of geneid_species" />
+        <conditional name="format_type">
+            <param name="out_format" type="select" label="Output format">
+                <option value="tabular" selected="true">Table</option>
+                <option value="csv">CSV</option>
+            </param>
+            <when value="tabular">
+                <param name="homologies" type="select" multiple="true" optional="false" display="checkboxes" label="Pair Types to keep:">
+                    <option value="one-to-one" selected="true">one-to-one</option>
+                    <option value="one-to-many" selected="true">one-to-many</option>
+                    <option value="many-to-one" selected="true">many-to-one</option>
+                    <option value="many-to-many" selected="true">many-to-many</option>
+                    <option value="paralogs" selected="true">Paralogs</option>
+                </param>
+            </when>
+            <when value="csv">
+                <param name="homologies" type="select" multiple="true" optional="true" display="checkboxes" label="Discard families with:">
```

Comment:
Isn't it a bit confusing that this multiselect has the opposite meaning of the one above? Maybe the intended usage is not clear to me.

Reply:
We want to remove all the families which has any of these type of homology, in tabular form we just want to remove pair.

Reply:
I still don't get why for tabular you ask which type to keep and for csv which type to discard.

Reply:
In tabular format families are shown as pairs, so we are able to filter on pairs, but in CSV format there are no pairs but user may want to filter out whole family where there is any of the selected homology type present